plugin/handlergen.go

```diff
@@ -28,6 +28,14 @@ func (p *OrmPlugin) generateDefaultHandlers(file *generator.FileDescriptor) {
 	}
 }
 
+func (p *OrmPlugin) generateAccoundIdWhereClause() {
```

Comment:
Per John D. should be AccountId instead of AccoundId, also appears where called in the code.

Reply:
Will fix this, thanks.